src/compiler/compile/render_dom/wrappers/IfBlock.ts

```diff
@@ -533,15 +618,14 @@ export default class IfBlockWrapper extends Wrapper {
 
 			const enter = b`
 				if (${name}) {
-					${dynamic && b`${name}.p(#ctx, #dirty);`}
-					${
-						has_transitions &&
-						b`if (${block.renderer.dirty(branch.dependencies)}) {
+					${dynamic && b`${name}.p(${if_ctx}, #dirty);`}
+					${has_transitions &&
+				b`if (${block.renderer.dirty(branch.dependencies)}) {
 							@transition_in(${name}, 1);
 						}`
-					}
+				}
```

Comment:
I think this end brace was better aligned previously

Reply:
i would hate to be @benmccann's cleaner.

Reply:
"You missed a spot!" :laughing:

Reply:
somehow i thought the linting would fix these